src/main/java/com/connexta/store/service/api/StoreService.java

```diff
@@ -54,4 +54,6 @@ void addIrm(
       @NotNull @Min(1L) @Max(10737418240L) final long fileSize,
       @Pattern(regexp = "^[0-9a-zA-Z]+$") @Size(min = 32, max = 32) final String datasetId)
       throws IndexDatasetException;
+
+  void promote(@NotNull String datasetId);
```

Comment:
✏️ I think we should use "unstage" instead of "promote"

Reply:
I like